act-rules-format.bs

```diff
@@ -240,7 +240,7 @@ The procedure of a rule MUST always return one of the following outcomes:
 
 - **Passed**: The Test Target passed the test
 - **Failed**: The Test Target failed the test
-- **Cannot tell**: It is unclear whether or not the Test Target passes of rails the test
+- **Undetermined**: The result of the test is undetermined. Human intervention or further testing is needed.
 
```

Comment:
Will update the statement to read more clearly, The result of the test is undetermined because Human intervention or further testing is needed.